Framework/CurveFitting/src/AugmentedLagrangianOptimizer.cpp

```diff
@@ -222,7 +221,6 @@ void AugmentedLagrangianOptimizer::minimize(std::vector<double> &xv) const {
         break;
     }
     if (ICM == 0.0) {
-      ret = FTolReached;
```

Comment:
Good spot, it's a large block of code! The behaviour seems unchanged, as `ret = Success;` [Old file, Line 210](https://github.com/mantidproject/mantid/pull/25213/files#diff-5a4677a945960304b4b1443a34c5cd27L210) was overwriting this in the next iteration anyway, and the `ret` value is only used in the `if` above.